benderliz/2021-12-9-FinalProjectSubmission.md

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
Could you rename this file path so that it begins with `_posts/` and ends up in the right directory? I also recommend lower-casing the keys `layout`, `title`, and `author` in your header.

Reply:
Hi Misha- I changed to _posts/ but submitted on accident before changing lower-casing. Not it is giving me a 404 error when I click the link to try to change it. I am wondering if the first PR needs to go through for me to create a new one to change layout/title/author?

Reply:
Ah, interesting. What happens if you refresh the page and then try to edit the file?